tests/tests-easy-digital-downloads.php

```diff
@@ -27,7 +27,9 @@ public function test_constants() {
 
 		// Plugin Folder Path
 		$path = str_replace( 'tests/', '', plugin_dir_path( __FILE__ ) );
```

Comment:
What about using `DIRECTORY_SEPARATOR` instead of `/`. That would make the `substr()` below unnecessary.